Form-Controls/styles.css

```diff
@@ -0,0 +1,28 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.form{
+    display: flexbox;
+    justify-items:self-start;
+    text-align: center;
+    border:20px solid  red ;
+    padding: 50px 50px;
```

Comment:
Hi @Amir200524 ,
Is there a reason for using px? I think px should only be used for elements with fixed size (e.g border, and box-shadow). I think rem is a better option.

For more context, below is a link 

https://www.shecodes.io/athena/28078-when-to-use-ems-rems-and-px-in-css